### DISCUSSION TITLE
Web Midi & Windows Troubleshooting

### DISCUSSION THREAD

**Discussion by jdlee022** [djipco/webmidi#242] [Q&A]
Hi! Has anyone ever run into trouble when trying to use your midi device with the browser and a DAW like Ableton at the same time? I don't have a problem on my macbook, but i can only connect to one at a time on my Windows PC.

**Comment by djipco** [comment]
I believe this is how MIDI works on Windows. See : 

https://tristancalderbank.com/2020/08/19/how-to-use-the-same-midi-device-on-windows-across-multiple-programs-at-the-same-time/amp/

**Reply by jdlee022** [reply]
Actually, i tried following that same article and it's still not working for me sadly
